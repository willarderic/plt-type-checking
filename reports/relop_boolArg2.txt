TOKEN: LPAREN  LEXEME: (					TOKEN: LPAREN    LEXEME: (
TOKEN: EQ  LEXEME: =						TOKEN: EQ        LEXEME: =
REL_OP -> =							REL_OP -> =
BIN_OP -> REL_OP						BIN_OP -> REL_OP
TOKEN: INTCONST  LEXEME: 0					TOKEN: INTCONST  LEXEME: 0
CONST -> INTCONST						CONST -> INTCONST
EXPR -> CONST							EXPR -> CONST
TOKEN: NIL  LEXEME: nil						TOKEN: NIL       LEXEME: nil
CONST -> nil							CONST -> nil
EXPR -> CONST							EXPR -> CONST
ARITHLOGIC_EXPR -> BIN_OP EXPR EXPR				ARITHLOGIC_EXPR -> BIN_OP EXPR EXPR
Line 1: Arg 2 must be integer or string				Line 1: Arg 2 must be integer or string
