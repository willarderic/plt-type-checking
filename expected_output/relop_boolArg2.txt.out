TOKEN: LPAREN    LEXEME: (
TOKEN: EQ        LEXEME: =
REL_OP -> =
BIN_OP -> REL_OP
TOKEN: INTCONST  LEXEME: 0
CONST -> INTCONST
EXPR -> CONST
TOKEN: NIL       LEXEME: nil
CONST -> nil
EXPR -> CONST
ARITHLOGIC_EXPR -> BIN_OP EXPR EXPR
Line 1: Arg 2 must be integer or string
